src/CoreBundle/Bridge/Chameleon/Migration/Script/update-1613996705.inc.php

```diff
@@ -0,0 +1,27 @@
+<h1>Build #1613996705</h1>
+<h2>Date: 2021-02-22</h2>
+<div class="changelog">
+    - ref #515: Wrong labels when editing spots of a static module
+</div>
+<?php
+
+$data = TCMSLogChange::createMigrationQueryData('cms_field_conf', 'en')
```

Comment:
Two is sufficient? There are 4 texts in the ticket description?

Reply:
actually no, but, the excepted texts are also incorrect. please see https://github.com/chameleon-system/chameleon-system/issues/515#issuecomment-794042438 issue comment